packages/fkweb/fk/forms.py

```diff
@@ -22,7 +22,7 @@ class UserChangeForm(forms.ModelForm):
 
     class Meta:
         model = User
-        fields = ('email', 'password', 'date_of_birth', 'is_active', 'is_superuser')
+        fields = ('password', 'date_of_birth', 'is_active', 'is_superuser')
 
     def clean_password(self):
         # Regardless of what the user provides, return the initial value.
```

Comment:
Could probably just do the same here for email. To make it readonly. And also set `email = forms.CharField(readonly=True)` or however it was. -- But this is very fine for now. We're not using this much. And might not for a whole lot longer. :P